pkg/scaffold/build_dockerfile.go

```diff
@@ -34,11 +34,24 @@ func (s *Dockerfile) GetInput() (input.Input, error) {
 	return s.Input, nil
 }
 
-const dockerfileTmpl = `FROM alpine:3.8
+const dockerfileTmpl = `# Binary builder image
+FROM golang:1.10.3 AS builder
```

Comment:
```suggestion
FROM golang:1.10.3-alpine3.8 AS builder
```

I think the base image in the builder should match our actual base image.

Also, maybe we should update to the latest go 1.10 (1.10.6)?